presto-benchto-benchmarks/src/test/resources/sql/presto/tpch/q13.plan.txt

```diff
@@ -8,9 +8,9 @@ remote exchange (GATHER, SINGLE, [])
                             final aggregation over (custkey)
                                 local exchange (GATHER, SINGLE, [])
                                     partial aggregation over (custkey)
-                                        join (LEFT, PARTITIONED):
-                                            remote exchange (REPARTITION, HASH, [custkey])
-                                                scan customer
+                                        join (RIGHT, PARTITIONED):
```

Comment:
Do we understand why this would change?

Reply:
This won't change actually. It's reverted it a while ago. These are the latest changes https://github.com/prestodb/presto/pull/14578/files

Reply:
Please do not introduce change and revert in later commit. I think for this PR, you can merge the two commits into one. If you want to keep 2 separate commits, please keep each one as only introducing the feature mentioned (one for adding null filter, one for introducing the session) and nothing else. Thanks!

Reply:
@rongrong I understand. I have now squashed all the changes in a single commit. 

Just so I can avoid this in the future, what's the recommended workflow in Presto to review multi-commit PRs? For example, pandas advises to accommodate code review changes in separate commits (https://pandas.pydata.org/docs/development/contributing.html#updating-your-pull-request) Apache Flink advises to squash everything in single amended commit (https://flink.apache.org/contributing/improve-website.html#submit-your-contribution) . What's the recommended way for Presto?